src/Contact.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+
+class Contact extends Component {
+    render() {
+        return (
+            <div>
+                <h5> Please fill out the contact form below to let us know what you think about Screw the Review! </h5>
+                <form name="contact" netlify>
+                    <p>
+                        <label>Name <input type="text" name="name" /></label>
```

Comment:
Where does the user actually submit their feedback (Don't see a box for it)? 

And what are we gonna do with the email and name info? Can't we get that from Auth0 in some form, if we want to collect it?

Reply:
Yep this is how you reply!

Reply:
@mackenzieleach